build/en/docs/index.njk

```diff
@@ -76,6 +76,15 @@ rootClass: docs
       Use the "Document" tab to control how your story gets published, add any important metadata and control whether pages auto-advance or need to be tapped manually.
     </p>
     <p class="img-wrap"><amp-img layout="intrinsic" width="148" height="800" src="./img/docs/11.png"></amp-img></p>
+    <p>
+      The "Checklist" tab has a pre-publish checklist. The pre-publish checklist alerts you about critical issues that may make your story ineligible to appear on platforms such as Google Discover. It offers recommendations about common accessibility shortcomings like low contrast and more.
+    </p>
+    <p class="img-wrap"><amp-img layout="intrinsic" width="480" height="356" src="./img/docs/checklist.png"></amp-img></p>
+    <h2 id="keyboard-shortcuts">Keyboard shortcuts &amp; navigation</h2>
+    <p>
+      View editor keyboard shortcuts by clicking the keyboard icon in the Workspace panel. Or press <code>ctrl</code> + <code>/</code> on Windows and <code>command ⌘</code> + <code>/</code> on Mac.
```

Comment:
Should we use [`<kbd>`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/kbd) for the keyboard shortcuts?

Reply:
TIL!